src/panels/Scene/SceneGraphNode/SceneGraphNodeView.tsx

```diff
@@ -0,0 +1,107 @@
+import { Box, Space, Tabs, Text } from '@mantine/core';
+
+import { useGetPropertyOwner, useGetVisibleProperties } from '@/api/hooks';
+import { PropertyOwner } from '@/components/PropertyOwner/PropertyOwner';
+import { useAppSelector } from '@/redux/hooks';
+import { PropertyOwner as PropertyOwnerType } from '@/types/types';
+
+import { SceneGraphNodeHeader } from './SceneGraphNodeHeader';
+import { SceneGraphNodeMetaInfo } from './SceneGraphNodeMetaInfo';
+
+interface Props {
+  uri: string;
+}
+
+export function SceneGraphNodeView({ uri }: Props) {
+  const propertyOwner = useGetPropertyOwner(uri);
+
+  if (!propertyOwner) {
+    throw Error(`No property owner found for SGN with uri: ${uri}`);
+  }
+
+  const propertyOwners = useAppSelector((state) => state.propertyOwners.propertyOwners);
+
+  // The SGN properties that are visible under the current user level setting
+  const visibleProperties = useGetVisibleProperties(propertyOwner);
+
+  const TabKeys = {
+    Renderable: 'Renderable',
+    Transform: 'Transform',
+    Other: 'Other',
+    Info: 'Info'
+  };
+
+  // We know that all scene graph nodes have the same subowners. However, not all of them
+  // are guaranteed to exist
+  const subowners: {
+    Renderable?: PropertyOwnerType;
+    Scale?: PropertyOwnerType;
+    Translation?: PropertyOwnerType;
+    Rotation?: PropertyOwnerType;
+  } = {};
+
+  propertyOwner.subowners.forEach((subowner) => {
+    const owner = propertyOwners[subowner];
+    if (owner) {
+      subowners[owner.identifier as keyof typeof subowners] = owner;
```

Comment:
Not a big fan of this, but unsure how to do it otherwise

Reply:
Yeah this is gnarly to read but we can define a type eg., `type TransformType = keyof typeof subowners` and do `subowners[owner.identifier as TransformType] = owner`